.github/CHANGELOG.md

```diff
@@ -1,12 +1,19 @@
 # Current develop
 
 ### New features
-  
+* Added two new internal CircuitComponent ``DsMap`` and `CftMap` to convert the representations. 
```

Comment:
This is all happenening in lab_dev, and we don't put it in the changelog. Just mention the fact that you added a triple